_posts/2019-06-26-cockpit-197.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 197
+author: kkoukiou
+date: 2019-06-26
+tags: cockpit linux
+slug: cockpit-197
+category: release
+summary: Machines new features
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 197.
+
+### Machines: Support all storage pool types for new disks
+
+A cockpit user can now add a disk to a VM either by creating a new volume or
+by choosing an existing volume which can reside on [any libvirt-supported pool
```

Comment:
Sorry.  Would go even a bit further: "by choosing an existing volume, which can now reside on"  (ie: add a comma and "now") to emphasize that this is the change.